pytorch_blade/bazel_build.py

```diff
@@ -42,6 +42,8 @@ def __init__(self, *args, **kwargs):
         self.test_suite = "//src:torch_blade_gtests"
         self.targets = [
             "@org_tensorflow//tensorflow/compiler/mlir/disc:disc_compiler_main",
+            "@org_tensorflow//tensorflow/compiler/mlir/xla/ral:libral_base_context.so",
+            "//src:libtorch_blade.so",
```

Comment:
nit: Target's from `@org_tensorflow` could be omitted if they were depended by targets of TorchBlade.